common/src/test/java/org/apache/falcon/util/ApplicationPropertiesTest.java

```diff
@@ -67,6 +67,12 @@ public void testResolveAlias() throws Exception {
         properties.resolveAlias();
         Assert.assertEquals(properties.getProperty(PROPERTY_1), PASSWORD_1);
         Assert.assertEquals(properties.getProperty(PROPERTY_2), PASSWORD_2);
+
+        // delete temporary jks files
```

Comment:
Can you please add it as @AfterClass to ensure these files are deleted even if there are test failures